openBCIBoard.js

```diff
@@ -708,14 +845,15 @@ function OpenBCIFactory() {
 
             if (!pInput && !nInput) {
                 this.impedanceTest.active = false;
-                this.writeOutDelay = k.OBCIWriteIntervalDelayMSShort;
+                //this.writeOutDelay = k.OBCIWriteIntervalDelayMSShort;
             } else {
-                this.writeOutDelay = k.OBCIWriteIntervalDelayMSLong;
+                //this.writeOutDelay = k.OBCIWriteIntervalDelayMSLong;
```

Comment:
your else isn't doing anything. I don't understand this section `if (!pInput && !nInput)`. does this mean that you're not testing? if so, why are these inputs being retrieved?